src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -482,7 +482,6 @@ std::vector<std::string> disabledTestPatterns() {
     // int8 / code-generation specific
     retVector.emplace_back(R"(smoke_LPT.*)");
     // Compressed weights are not supported
-    retVector.emplace_back(R"(smoke_MatMulCompressedWeights.*)");
     retVector.emplace_back(R"(smoke_MatMulSharedCompressedWeights.*)");
```

Comment:
Should shared compressed weights be supported automatically by the PR changes? If yes, can we enable the corresponding tests?

Reply:
It should be. However I would prefer to enable these tests only once weights decompression fucntionality will be landed as default one on ARM. For now only MatMulCompressedWeights makes sense for future development activities.